### ISSUE TITLE
Share via query string

### ISSUE THREAD

**Issue by james2doyle** [moroshko/accessible-colors#74]
Would anyone be interested in the ability to share the current page state in the query string? 

This would allow you to share the page and come back to it as it was when you filled out the form.

I am offering to work on this feature if it is useful to anyone

Example: https://accessible-colors.com/?color=333333&size=18&format=px&weight=regular&bg=eeeeee&design=aa
Or: https://accessible-colors.com/?color=bada55&size=22&format=pt&weight=bold&bg=000&design=aaa

**Comment by vedranio** [comment]
@james2doyle yes, this is something that a few people have asked about. It would be a useful feature

**Comment by james2doyle** [comment]
Opened #75
